test/data/allele_slot_annotation_ingest_test.json

```diff
@@ -79,6 +99,19 @@
           ],
           "internal": false
         }
+      ],
+      "allele_synonym_dtos": [
+        {
+          "name_type_name": "systematic_name",
```

Comment:
should be "synonym" no?

Reply:
My understanding was that synonym could be any type of name.

Reply:
There are no rules in place for name_type for `AlleleSymbolSlotAnnotation` (or the DTO equivalents), and IMO 'systematic_name' would be a valid name type - in the case of Alleles there is no other slot for systematic names (if anyone has them), and in the case of genes you may want to add a systematic name that is not the one current systematic name that would go under `GeneSystematicNameSlotAnnotation`.

Also, based on the `name_type` slot definition in core.yaml, I don't believe that 'synonym' is one of the proposed name types (although based on the list in the description there is a discrepancy between 'nomenclature_symbol' used in the slot description and 'symbol' used in the rules/notes for the SymbolSlotAnnotation classes.

Reply:
@chris-grove - yes, synonym can be of any type. Can have old "full_name" or "systematic_name" synonyms in here.
@markquintontulloch - I agree that "systematic_name" can be valid for symbol (for cases where gene or allele has not yet been named):e.g., FB has systematic names for genes and alleles where no symbol exists. I've updated the `SymbolSlotAnnotationDTO`, `AlleleSymbolSlotAnnotation` and `GeneSymbolSlotAnnotation` objects accordingly.